CLAUDE.md

```
@@ -0,0 +1,170 @@
+// 例2: 名前のみ
+IdentityStringGenerator::generate('', 'name')
+// 結果: "#name"
+
+// 例3: インターフェースのみ（名前はName::ANYがデフォルト）
+IdentityStringGenerator::generate(FakeRobotInterface::class)
+// 結果: "NaokiTsuchiya\RayDiPsrContainer\FakeRobotInterface#"
+```
+
+### Ray.DI Integration Pattern
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)